### ISSUE TITLE
Support for .Net Standard 2.0

### ISSUE THREAD

**Issue by tb-mtg** [contre/Windows-API-Code-Pack-1.1#16]
Could you please add support for .Net Standard/.Net Core 2.0 ?

**Comment by contre** [comment]
This could be done but it will necessitate removing a variety of things because they are dependent on the windows UI frameworks which are not supported under Core

**Comment by dahall** [comment]
@contre I have forked the code and have upgraded all the projects to the new SDK format and made them support .NET Core 3.0. If I submit PRs, are you in a position to merge them and release updated NuGet packages?

**Comment by contre** [comment]
My wife and I just had our second child and as such I will not be able to do so for the near future. I can certainly review a pull request but I am pretty limited in computer access at the moment.

**Comment by czdietrich** [comment]
Is there any progress here? 
I would like to avoid making another fork of this source code, as there are already several forks and NuGets, but none of them seem to support .NET Core.

@dahall Can you create the pull request, so it can be reviewed?

**Comment by dahall** [comment]
I'm happy to create a PR, however I must warn everyone that I also ran code cleanup against all the files so there are 300 file changes. Almost no code logic is changed, other than those required for .NET Framework vs. .NET Core requirements. All the *.csproj files changed dramatically. I have done only minimal testing of the new builds. Still want my PR?

**Comment by contre** [comment]
I created one from your branch already so there was at least interest from me.

**Comment by dahall** [comment]
It is done. PR #22 has been created and I've included some notes on how to resolve conflicts.

**Comment by contre** [comment]
This has now been fixed. Reopen or report a new issue if you have problems.

Still waiting on generating new nuget packages since additional changes are being made

**Comment by GreatFireWall** [comment]
@dahall Could you please push the new version to NuGet ?

**Comment by dahall** [comment]
@GreatFireWall I don't have permissions

**Comment by contre** [comment]
It would be me that needs to do that. I'll do so once #24 is merged

**Comment by GreatFireWall** [comment]
Got it, thank you all !

**Comment by timcassell** [comment]
.Net Standard support wasn't added, only .Net Core was. Is .Net Standard possible?

**Comment by czdietrich** [comment]
I think, it is not really possible. These projects use `UseWPF` and `UseWindowsForms` switches which are only available in .NET Core and .NET Framework, but not in .NET Standard.

**Comment by timcassell** [comment]
I think it could be done with `#if !NETSTANDARD2_0` conditional compilation blocks, so the parts that don't rely on WPF/WinForms can be used. I was able to extract at least a bit of that in https://github.com/dotnet/BenchmarkDotNet/pull/2158.
